.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Changelist
+<!-- Bulleted list of changes made -->
+- 
+
+## Breaking Changelist
+<!-- Delete section if none, bulleted list of breaking changes -->
+- 
+
+## Issue number/link
+<!-- Link to related issues -->
+
+## Test plan
+<!-- How will this be tested -->
```

Comment:
I think tests for a feature/fix of a PR should go into the PR, maybe reword this to "how was this tested" or combine it with changelist?

Reply:
Agree with @noot. Every PR should be accompanied by tests (or good arguments why tests don't apply), which is probably what is meant by this section.

But it should be rephrased so we make it clear that this must not be punted to a future PR.